src/mmrelay/plugin_loader.py

```
@@ -870,62 +923,548 @@ def _check_auto_install_enabled(config):
+        bool: `True` if a default branch was successfully checked out and pulled, or `True` if no branch could be checked out and the repository's current state is retained.
+    """
+    for default_branch in default_branches:
+        try:
+            _run_git(["git", "-C", repo_path, "checkout", default_branch], timeout=120)
+            _run_git(
+                ["git", "-C", repo_path, "pull", "origin", default_branch], timeout=120
+            )
+            logger.info(
+                f"Using {default_branch} instead of {ref_value} for {repo_name}"
+            )
+            return True
```

Ruff (0.14.4)

Consider moving this statement to an `else` block

(TRY300)